docs/notary/setup-Komodo-Notary-Node.md

```diff
@@ -939,28 +1047,28 @@ Here is an example of a Main Server start script that will start Notary easy min
 ```bash
 #!/bin/bash
 source ~/komodo/src/pubkey.txt
-bitcoind &
-~/hush3/src/hushd -pubkey=$pubkey &
-~/VerusCoin/src/verusd -pubkey=$pubkey &
+
+bitcoind -deprecatedrpc=estimatefee &
+/home/$USER/VerusCoin/src/verusd -pubkey=$pubkey &
+/home/$USER/komodo/src/komodod -gen -genproclimit=1 -notary -pubkey=$pubkey -minrelaytxfee=0.000035 -opretmintxfee=0.004 &
+/home/eclips/dexkomodo/src/komodod -ac_name=DPOW -dexp2p=2 -addnode=136.243.58.134 -pubkey=$PUBKEY3RD -handle=$NAME &
```

Comment:
please change this path since it contains your username and change `PUBKEY3RD` to `pubkey`
(btw imo it's more compatible/clear way to use $HOME/ or ~/ as it's using in other places of this doc instead of /home/$USER/ because userhome might be not in /home folder )